### DISCUSSION TITLE
Error on Italian Model (stanford-corenlp-4.4.0-models-italian.jar)

### DISCUSSION THREAD

**Discussion by ClaudeHu** [stanfordnlp/CoreNLP#1258] [General]
After I downloaded the stanford-corenlp-4.4.0-models-italian.jar, I tried to run the Italia model with Stanford CoreNLP API in python: 

```python
from pycorenlp import StanfordCoreNLP
params = {'outputFormat': 'json', 'outputDirectory': outputDir, 'replaceExtension': True}
CoreNLP_nlp = subprocess.Popen(
              ['java', '-mx' + str(5) + "g", '-cp', os.path.join(CoreNLPdir,'*'), 'edu.stanford.nlp.pipeline.StanfordCoreNLPServer','-props', 'italian', '-timeout', '999999'])
nlp = StanfordCoreNLP('http://localhost', port=9000)
CoreNLP_output = nlp.annotate(text, properties=params)
```

However, it seems that the dependencies in the output is different from the dependencies annotated in the online demo (https://corenlp.run)

**Comment by AngledLuffa** [comment]
Would you give an example where the output is different?

**Reply by ClaudeHu** [reply]
In this example sentence: 
```
Ci sono tanti, più giovani di te o meno giovani, che vivono in attesa d'esperienze straordinarie; dai libri, dalle persone, dai viaggi, dagli avvenimenti, da quello che il domani tiene in serbo. Tu no. Tu sai che il meglio che ci si può aspettare è di evitare il peggio.
```
The syntactical head / governor of the token "sono" is "tanti", and the dependency relation is "cop". However, in the output of Stanford CoreNLP running on API in python, "sono" is the ROOT.

**Comment by AngledLuffa** [comment]
Got it.  It seems the python client is accidentally adding a few annotators
which aren't appropriate for Italian.  I'll look into it

**Comment by AngledLuffa** [comment]
Actually, while I happened to notice the annotator issue and have been working on fixing it, when I run that particular example I still get the expected head.

From the basic dependencies:

```
    edge {
      source: 3
      target: 2
      dep: "cop"
      isExtra: false
      sourceCopy: 0
      targetCopy: 0
      language: UniversalEnglish
    }
```

and down at the bottom:

```
    root: 3
```

I suppose labeling the language "UniversalEnglish" also needs fixed.
